README.md

```diff
@@ -1,4 +1,28 @@
-Disclaimer: suboptimal in many ways, but more than enough for what I need.
+# Branch search
+
+A simple terminal branch search for git
+
+## Install
```

Comment:
Given that we have this sections now, we can mention `ncurses` is the only non-standard dependency and how it can be installed.